src/batch-client/index.ts

```diff
@@ -564,6 +566,16 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		});
 
+	public getWorkingHours = ({ start, end, names }: WorkingHoursOptions) =>
+		this.jsonRequest({
+			name: 'GetWorkingHours',
+			body: {
+				s: start,
```

Comment:
use denormalize function instead of writing manual mapping

Reply:
done